src/tests/ftest/util/dmg_utils.py

```diff
@@ -406,7 +423,19 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
         # },
         # "error": null,
         # "status": 0
-        output = self._get_json_result(("pool", "create"), **kwargs)
+        output = self._get_json_result(("pool", "create"),
+                                       json_err=True, **kwargs)
+        if output["error"] is not None:
+            self.log.error(output["error"])
+            if output["status"] in RETRYABLE_POOL_CREATE_ERRORS:
+                time.sleep(POOL_RETRY_INTERVAL)
+                return self.pool_create(scm_size, uid=uid, gid=gid,
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'group' in method call